src/app/models/auth-data.model.ts

```diff
@@ -0,0 +1,6 @@
+export interface AuthData {
```

Comment:
It would be better to put all the login related models and services inside the same folder  `src/app/components/login/`.